Cargo.toml

```diff
@@ -1,3 +1,7 @@
+[workspace]
```

Comment:
I feel like there was some reason we didn't do this before, but I don't know what it could be.

Reply:
`maturin` pyo3 doesn't like to be built as part of a workspace, but adding `wheel` to the exclude key makes it work